pages/en/network/indexing.mdx

```diff
@@ -672,31 +674,33 @@ indexer indexer actions cancel
 
 indexer indexer actions approve 1 3 5
 
-indexer indexer actions execute approve 
+indexer indexer actions execute approve
 ```
 
-Note that supported action types for allocation management have different input requirements:
-`Allocate` - allocate stake to a specific subgraph deployment
+Note that supported action types for allocation management have different input requirements: `Allocate` - allocate stake to a specific subgraph deployment
```

Comment:
Is this Prettier's doing? I think to avoid it, the line that starts with `Allocate` should be a list item (so prepend `- ` to it), and same thing with `Unallocate` and `Reallocate`. Then we should nest everything below them one more level (2 extra spaces).

Reply:
Yes, I think so - good call! Thank you!